app/src/main/java/ch/epfl/sdp/Event.java

```diff
@@ -21,7 +21,7 @@
     private final String mAddress;
     private final LatLng mLocation;
     private final String mOrganizerRef;
-    static private SimpleDateFormat mFormatter = new SimpleDateFormat("dd/MM/yyyy", Locale.getDefault());
+    static private SimpleDateFormat mFormatter = new SimpleDateFormat("dd/MM/yyyy kk:mm", Locale.getDefault());
```

Comment:
Just fyi, `kk:mm` puts midnight as 24:00, `HH:mm` puts midnight as 00:00, use the one you think is best here.

Reply:
ok